src/navigators/Navigator.tsx

```diff
@@ -67,7 +68,7 @@ class Navigator extends React.Component<Props, State> {
     fetchBlockHeight();
     this.initLanguage();
 
-    if (!__DEV__) {
+    if (!__DEV__ || !isEmulator()) {
```

Comment:
It's working for sure? `isEmulator()` returns Promise so it won't be a valid condition as far as I understand

Reply:
Yes, you're right, it was working on my side because of returned promise, I've didn't checked what's actually was returned. Thank you!